src/widget/wmainmenubar.cpp

```diff
@@ -599,6 +607,23 @@ void WMainMenuBar::initialize() {
     addMenu(pHelpMenu);
 }
 
+WMainMenuBar::~WMainMenuBar() {
+    qDebug() << "Delete WMainMenuBar";
+
+    delete m_bShowMenuBar;
+}
+
+void WMainMenuBar::showHideMenuBar(double v) {
+    if (v > 0) {
+        int minHeight = sizeHint().height();
```

Comment:
Does it make sense to read this after initialization, or keep it here in case fonts or decoration or whatever is changed by OS measures while Mixxx runs?